packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/quanticGeneratedAnswerContent.js

```diff
@@ -0,0 +1,68 @@
+import MARKED_JS from '@salesforce/resourceUrl/marked';
+import {transformMarkdownToHtml} from 'c/quanticUtils';
+// @ts-ignore
+import {loadScript} from 'lightning/platformResourceLoader';
+import {LightningElement, api} from 'lwc';
+// @ts-ignore
+import generatedMarkdownContentTemplate from './templates/generatedMarkdownContent.html';
+// @ts-ignore
+import generatedTextContentTemplate from './templates/generatedTextContent.html';
+
+export default class QuanticGeneratedAnswerContent extends LightningElement {
+  @api isStreaming;
+  @api answerContentFormat = 'text/plain';
+  @api
+  get answer() {
+    return this._answer;
+  }
+  set answer(value) {
+    this._answer = value;
+    if (this.answerContentFormat === 'text/markdown') {
+      this.updateHtmlContent();
+    }
+  }
+
+  _ready = false;
+  _answer;
+
+  connectedCallback() {
+    if (this.answerContentFormat === 'text/markdown') {
```

Comment:
The component doesn't stay "connected" in between rendering of different answers.

Since the `QuanticGeneratedAnswer` component doesn't render the `QuanticGeneratedAnswerContent` component when there's no answer, or when you change your query to a different one, the `QuanticGeneratedAnswerContent` component completely gets disconnected and re-connected in the page.